examples/api/lib/widgets/page_view/page_view.1.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for a [PageView] using the `findChildIndexCallback` argument
```

Comment:
Nit: Add a period at the end of the comment.

Style guide ref: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar